### ISSUE TITLE
Headers Stay stuck after reposition

### ISSUE THREAD

**Issue by AvogadroSG1** [mkoryak/floatThead#348]
Hello!

We use JQuery 3.1.1 and Bootstrap 3.  Before updating to JQuery 3.1.1 this worked fine.

Problem is caused when the window is resized.  It used to automatically recalculate the headers (I assumed did an automatic reflow).  Now the headers become stuck and don't move anytime after it.

My current workaround is:

```
              var resizeTimer;
                $(window).resize(function () {
                    clearTimeout(resizeTimer);
                    resizeTimer = setTimeout(reinitFixedHeaders, 50);
                });

                function reinitFixedHeaders() {
                    $(".fixed-header").floatThead('destroy');
                    $(".fixed-header").floatThead({ zIndex: 60 });
                }
```

Fiddle:  https://jsfiddle.net/avogadrosg1/hrr4b2ob/1/

Browsers:  IE 11, FireFox 50.1
OS:  Windows 8.1 and 10
JQuery 3.1.1
Newest FloatThead

Edit: Removed some text

**Comment by mkoryak** [comment]
This is a bug. I'll fix it in the next 24 hours and release a new version

**Comment by mkoryak** [comment]
fixed in 2.0.0
https://github.com/mkoryak/floatThead/releases/tag/2.0.0

your fiddle magically works
